### PR TITLE
Ьultiline cloudConfig support

### PR DESCRIPTION
**Pull request by aleiner**
It is currently not possible to set a multi-line cloud-config to be passed through to the created hosts.  Updated the template to allow for multi-line, and formatting, as well as updated the usage comment.

### CODE REVIEW

File: charts/values-vsphere.yaml
Comment (snasovich) [inline charts/values-vsphere.yaml:71]: Can we continue using the previous format? The comment seems to state we have to use multi-line format while the example value `cloudConfig: ""` is not technically following this format.

**Review by snasovich — COMMENTED** [review]
Have one question / suggestion

File: charts/values-vsphere.yaml
Comment (aleiner) [reply charts/values-vsphere.yaml:71]: PR update by removing old example.

**Review by snasovich — APPROVED** [review]
LGTM, need one more approval before merging.

**Review by jakefhyde — APPROVED** [review]

**Pull request merged**
